src/plugins/controls/public/index.ts

```diff
@@ -12,36 +12,41 @@ export type {
   ControlGroupApi,
   ControlGroupRuntimeState,
   ControlGroupSerializedState,
+  ControlStateTransform,
+  ControlPanelState,
 } from './react_controls/control_group/types';
+export {
+  controlGroupStateBuilder,
+  type ControlGroupStateBuilder,
+} from './react_controls/control_group/utils/control_group_state_builder';
+
 export type {
   DataControlApi,
   DefaultDataControlState,
   DataControlFactory,
   DataControlServices,
 } from './react_controls/controls/data_controls/types';
-export { controlGroupStateBuilder } from './react_controls/control_group/control_group_state_builder';
+export type { OptionsListControlState } from './react_controls/controls/data_controls/options_list_control/types';
 
-/**
- * TODO: remove all exports below this when control group embeddable is removed
- */
+export { ACTION_EDIT_CONTROL } from './react_controls/actions/edit_control_action/edit_control_action';
 
-export type {
-  ControlOutput,
-  ControlFactory,
-  ControlEmbeddable,
-  ControlEditorProps,
-  CommonControlOutput,
-  IEditableControlFactory,
-  CanClearSelections,
-} from './types';
+export {
+  ACTION_DELETE_CONTROL,
+  ControlGroupRenderer,
```

Comment:
Can you export a lazy loaded `ControlGroupRenderer`. This should shrink the bundle size even more. 

Here is an example from maps

Reply:
Good call - done in 15d2d29bd296dc9ef7ebab60524094058e8d3f54